applications/ecallmgr/src/ecallmgr_call_control.erl

```diff
@@ -1172,55 +1250,36 @@ get_keep_alive_ref(#state{keep_alive_ref=TRef
 %%
 %% @end
 %%--------------------------------------------------------------------
--spec bind_to_events(atom(), ne_binary()) -> 'true'.
-bind_to_events(Node, CallId) ->
+-spec bind(atom(), ne_binary()) -> 'true'.
+bind(Node, CallId) ->
     lager:debug("binding to call ~s events on node ~s", [CallId, Node]),
     'true' = gproc:reg({'p', 'l', {'call_event', Node, CallId}}),
-    'true' = gproc:reg({'p', 'l', {'event', Node, <<"CHANNEL_CREATE">>}}),
-    'true' = gproc:reg({'p', 'l', {'event', Node, <<"CHANNEL_DESTROY">>}}).
+    'true' = gproc:reg({'p', 'l', ?LOOPBACK_BOWOUT_REG(CallId)}),
+    'true'.
```

Comment:
Since the previous line must match 'true', no need to end with a bare 'true' here.